common/src/Renderer/PrimitiveRenderer.cpp

```diff
@@ -75,7 +78,9 @@ bool PrimitiveRenderer::LineRenderAttributes::operator<(
 void PrimitiveRenderer::LineRenderAttributes::render(
   IndexRangeRenderer& renderer, ActiveShader& shader) const
 {
-  glAssert(glLineWidth(m_lineWidth));
+  QScreen* screen = QApplication::screens().at(0);
```

Comment:
It would be better to use `QWindow::devicePixelRatio` because you are just making a guess as to which monitor TB's window will be on. The renderer should also not depend on Qt.

A better solution would be to pass `devicePixelRatio` into `RenderContext`. `RenderContext` is constructed in `MapViewBase::doRender`, and there, you can access `QWindow::devicePixelRatio` to use the correct value for the view being rendered.